tests/contracts/wallet_2.scilla

```diff
@@ -14,11 +14,23 @@ let mk_contract_initialized_event =
 (* Event for communicating a new transaction id *)
 let mk_transaction_added_event =
   fun (tc : Uint32) =>
-    { _eventname : "Transaction created" ; transactionId : tc }
+  fun (recipient : ByStr20) =>
+  fun (amount : Uint128) =>
+  fun (tag : String) =>
+    { _eventname : "Transaction created" ; transactionId : tc; recipient : recipient; amount : amount; tag : tag  }
 
-(* Event for communicating the signing of a transaction *)
+(* Event for communicating the execution of a transaction *)
 let mk_transaction_executed_event =
-    { _eventname : "Transaction executed" }
+  fun (recipient : ByStr20) =>
+  fun (amount : Uint128) =>
+  fun (tag : String) =>
+    { _eventname : "Transaction executed"; recipient : recipient; amount : amount; tag : tag }
+
+(* Event for communicating that a transaction was signed *)
+let mk_signed_transaction_event =
+  fun (tc : Uint32) =>
+  fun (signedBy : ByStr20) =>
+    { _eventname : "Transaction signed"; transactionId : tc; signedBy : signedBy }
```

Comment:
remove signedBy

Reply:
do you want that removed or renamed? (to say `_sender`)?

Reply:
Remove it completely.

Reply:
As this information is already available from the transaction. @vaivaswatha

Reply:
> As this information is already available from the transaction. @vaivaswatha

@AmritKumar is already done.